frontend/Views/ForumFeed/index.js

```diff
@@ -81,9 +92,20 @@ class ForumFeed extends Component {
       sortingMethod,
       error,
       searchInput,
-      filteredDiscussions
+      filteredDiscussions,
+      authenticated
     } = this.props;
 
+    // TODO: reduce duplicate code
```

Comment:
will remove this comment, this isn't duplicate anymore (could clean it up anyways though, wrote this very quickly)